recipes/docker.rb

```diff
@@ -17,9 +17,11 @@
 # limitations under the License.
 #
 
-docker_service 'default' do
-  #tls false
-  #provider Chef::Provider::DockerService::Sysvinit
-  action [:create, :start]
-  group 'wheel'
+if node['os_version'] !~ /^3\.2\.0.*$/
```

Comment:
Originally this was for Ubuntu 12...
but yeah - if you have outdated kernel - docker won't fly